backend/infrahub/core/branch/tasks.py

```diff
@@ -298,7 +305,12 @@ async def create_branch(model: BranchCreateModel, service: InfrahubServices) ->
             registry.branch[obj.name] = obj
             await service.component.refresh_schema_hash(branches=[obj.name])
 
-        event = BranchCreateEvent(branch=obj.name, branch_id=str(obj.uuid), sync_with_git=obj.sync_with_git)
+        event = BranchCreatedEvent(
+            branch_name=obj.name,
+            branch_id=str(obj.uuid),
```

Comment:
This part stood out to me. I noticed that we're using `obj.get_id()` on the Rebase and Delete event but `str(obj.uuid)` here. It feels inconsistent. I think one of them is the internal database ID within Neo4j.

Reply:
Good catch